database/seeds/PhotoTableSeeder.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class PhotoTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        self::deleteDirectory('public/images/photoFromRaspi');
+
+        self::createDir('public/images');
+        self::createDir('public/images/photoFromRaspi/');
+        self::createDir('public/images/photoFromRaspi/' . date('Y') . '/');
+        self::createDir('public/images/photoFromRaspi/' . date('Y') . '/' . date('m'));
+
+        factory(App\Photo::class, 20)->create();
+    }
+
+    private static function createDir($path)
+    {
+        if (!file_exists($path)) {
+            mkdir($path, 0700);
+        }
+    }
+
+
+    private static function deleteDirectory($dir)
```

Comment:
https://laravel.com/docs/5.6/filesystem#deleting-files

Reply:
Please remove redundant code and use laravel framework functions instead. The same for storing files

Reply:
added